src/app/core/components/left-header/left-header.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="nav-header" *ngIf="!compactMode">
   <div class="nav-control">
-    <span class="nav-collapse" (click)="hideLeftMenu()" tabindex="0">
+    <span class="nav-collapse" (click)="hideLeftMenuAndHeaders()" tabindex="0">
       <!-- <i class="fa fa-bars"></i> -->
       <img src="assets/images/menu-open.svg" style="width: 2rem; height: 2rem;" />
     </span>
```

Comment:
If now it is just a full screen button, let's just use a full screen icon?
https://fontawesome.com/v5.15/icons/expand-alt?style=solid or https://fontawesome.com/v5.15/icons/expand-arrows-alt?style=solid
and https://fontawesome.com/v5.15/icons/compress-alt?style=solid

Reply:
Used the expand-arrows-alt and compress-arrows-alt, thanks for the suggestion.